tools/AndroidDependencies/.gitignore

```diff
@@ -0,0 +1,3 @@
+Library/
+Packages/
```

Comment:
good points!
both Packages and meta probably shouldn't be in that gitignore?

However, Library, Temp and obj should be.

Reply:
It depends how far we want to have in the "ignore all the things" slider:

baseline would be `Library`, `Temp`, and `obj`
next level would add ProjectSettings (minus project version) and Packages
maximum would be "everything (including meta files) but this .cs file"

Reply:
Packages not really, right? because if we actually want to add Packages, we need them synchronized via git

Reply:
For this PR I don't think any packages would be necessary unless we may need to do some package configuration later on

Reply:
yes, we don't need it right now. But why ignore it, if it might be necessary later. I don't see a reason to ignore the folder :)

Reply:
> The meta files are in the .gitignore because they are ignored by default from the top-level .gitignore (which I believe utilises some template).

Which file specifically?

Reply:
they sometimes do created and sometimes don't. Maybe Firtina knows the rules on when they get created.

Reply:
I'll do some experiments on this branch :)

Reply:
@firtina-improbable https://github.com/spatialos/gdk-for-unity-shared-ci/blob/master/.gitignore this one

Reply:
I guess we can just remove the ignore from there :)

Reply:
".meta" : Ah I see, sorry i was looking at the wrong repo's gitignore file 🤦‍♂️